src/home.css

```
@@ -3,36 +3,254 @@
     padding: 0;
     margin: 0;
 }
+
+:root {
+    --main-bg-color: #E6EBEF;
+    --primary-color: #00A6C8;
+    --grey-lighter: #B3C1CD;
+    --grey-blue: #335978;
+    --red: #E30F3D;
+    --orange: #FF9D43;
+    --green: #7DB13A;
+    --grey-darker: #252828;
+    --grey-white: #EDEDED;
+    --white: #FFFFFF;
+    --grey:#868686;
+}
+
 body {
-    background-color:#E6EBEF;
+    background-color: var(--main-bg-color);
 
-    width: 100%;
-    min-height: 100vh;
+    width: 100vw;
+
+    color: var(--grey-darker);
 
     font-family: "Gotham", "Arial", sans-serif;
+    font-weight: 800;
+    font-style: normal;
 
     display: flex;
-    align-items: center;
+}
+
+.menu {
+    width: 64px;
+
+    background-color: var(--grey-darker);
+}
+
+.content {
+    width: 100%;
+    height: 100%;
+
+    background-color: var(--grey-white);
+}
+
+.navbar-top {
+    width: 100%;
+    height: 60px;
+
+    background-color: var(--white);
+}
+
+.navbar-container {
+    width: 100%;
+    display: flex;
+    /* align-items: center; ceci ne marche pas du coup j'ai mit un padding */
+    padding-top: 23px;
     justify-content: center;
+
+    font-style: normal;
+    font-weight: 400;
+
+    color:var(--grey);
+}
+
+.navbar-container p{
+    padding-right: 32px;
+}
+.text-style {
+    font-weight: 500;
+    font-style: normal;
+
+    color :var(--grey-darker); 
 }
+
+.main {
+    width:calc(100% - 60px);
```

```suggestion
    width: calc(100% - 60px);
```

Fais attention à toujours mettre un espace après les `:`, c'est plus facile à lire